File: Day1/Findcharactercase.py
def Findcharactercase(a):
    if a.isupper():
        return '1'
    elif a.islower():
        return '0'
    else:
        return '-1'
ab = input()
print(Findcharactercase(ab))    

# Another way to solve
# fcc=input()
# if fcc.isupper():
#     print('1')
# elif fcc.islower():
#     print('0')
# else:
#     print('-1')

File: Day1/Findcharactercase.c
#include <stdio.h>
#include <ctype.h>

int Findcharactercase(char a)
{
    if(isupper(a))
    {
        return 1;
    }
    else if (islower(a))
    {
        return 0;
    }
    else
    {
        return -1;
    }
}

int main()
{
    char charinput;
    printf("Enter the character \t");
    scanf("%c",&charinput);
    // printf("Entered character is -> %c \n ",charinput);

    int result = Findcharactercase(charinput);
    printf("%d",result);    
}
    













// int main()
// {
// char inp;
// scanf("%c",&inp);
// if (isupper(inp)) {
//     printf("f");
// }

// }